book.bib

```diff
@@ -144,7 +134,7 @@ @book{Bran1995
 @misc{Broc2019,
   author = {Jon Brock},
   year = {2019},
-  title = {"A love letter to your future self": What scientists need to know about FAIR data},
+  title = {A Love Letter to Your Future Self: What Scientists Need to Know About {FAIR} Data},
```

Comment:
I think the quotation marks are in the title of the original?

Reply:
Yeah, I wasn't sure what to do here. There are quotation marks in the original title, but our referencing style in the PDF puts quotation marks around the title of articles, so it ends up formatting with an open and closed quotation at the beginning (i.e. ""A love...) and then an open quotation after self and a closed at the end...

Reply:
What about using single quotes?

Reply:
I believe the double double-quotes are canonical, but yeah, it looks weird, so let's go with single quotes just to give future bibliographic analysis software a heart attack.